modules/apps/store/extensions/assets/mobileapp/pages/assets.jag

```diff
@@ -93,7 +93,46 @@ require('/modules/store.js').exec(function (ctx) {
                 }
```

Comment:
Comments given in https://github.com/wso2/product-app-manager/pull/78. apply to here as well.